Can't resolve '~storybook'

<!-- **IMPORTANT!**
Please make sure to look for an answer to your question in our documentation and the documentation before asking a question here.

If you have a general question regarding the module use Discord `modules` channel. Thanks!

Nuxt Discord: https://discord.nuxtjs.org/
-->


Hello,

When i create preview.js file & inject this line `import '~/.nuxt-storybook/storybook/preview.js'
`  , i have this error : 
`
ERROR in ./.nuxt-storybook/storybook/preview.js
Module not found: Error: Can't resolve '~storybook' in 
`

What is the target of `~storybook` ? and why i have this error ? 

Here my configuration : 
```
// main?.
const path = require('path');

module.exports = {
    webpackFinal(config, options) {
        config.module.rules.map((rule) => {
            if (rule.test instanceof RegExp && rule.test.test('.scss')) {
                rule.use = [
                    ...rule.use,
                    {
                        loader: require.resolve('sass-resources-loader'),
                        options: {
                            resources: [
                                path.resolve(__dirname, '../assets/css/core/_global.scss'),
                                path.resolve(__dirname, '../assets/css/core/main.scss')
                            ]
                        }
                    }
                ]
                return rule;

            }
            return rule;
        });

        config.resolve.alias = {
            ...config.resolve.alias,
            '@': path.resolve(__dirname, '../')
        }
        return config;
    },
    stories: [
        '../components/**/*.stories.@(ts|js|mdx)',
        '../storybook/**/*.stories.@(ts|js|mdx)',
    ],
    addons: [
        '@storybook/addon-docs',
        '@storybook/preset-scss',

]
}

```

```
// preview.js
import Vue from 'vue';
import Color from '@/storybook/colors/Color.vue'
import '~/.nuxt-storybook/storybook/preview.js'

Vue.component('Color', Color);

```

Thanks for answers.
Have a good day.

--- a/renovate.json5
+++ b/renovate.json5
@@ -9,13 +9,13 @@
     // Use "chore" type for all commit messages
     ':semanticCommitTypeAll(chore)',
     // Pin all dev dependencies following https://docs.renovatebot.com/dependency-pinning/
-    ':pinDevDependencies'
+    ':pinDevDependencies',
   ],
   packageRules: [
     {
       // Pin all docs dependencies following https://docs.renovatebot.com/dependency-pinning/
-      "matchFileNames": ["docs/package.json"],
-      "rangeStrategy": "pin"
+      matchFileNames: ['docs/package.json'],
+      rangeStrategy: 'pin',
     },
     {
       // Bump all @storybook/* packages so that always the latest version is used

--- a/renovate.json5
+++ b/renovate.json5
@@ -8,8 +8,15 @@
     'schedule:weekends',
     // Use "chore" type for all commit messages
     ':semanticCommitTypeAll(chore)',
+    // Pin all dev dependencies following https://docs.renovatebot.com/dependency-pinning/
+    ':pinDevDependencies'
   ],
   packageRules: [
+    {
+      // Pin all docs dependencies following https://docs.renovatebot.com/dependency-pinning/
+      "matchFileNames": ["docs/package.json"],
+      "rangeStrategy": "pin"
+    },
     {
       // Bump all @storybook/* packages so that always the latest version is used
       matchPackagePatterns: ['storybook', '@storybook/*'],

